### ISSUE TITLE
Mapping Strings from OPCUA server with basyx-databridge to AAS v3

### ISSUE THREAD

**Issue by janbartenbach** [eclipse-basyx/basyx-databridge#257]
Hi,
when using the databridge with an AAS v3 it was only possible to me to bridge non-String type values.
Using the Demo Setup from basyx-applications/opc2aas, with some adjustments (aasv3-endpoint on a submodel server and the "api" field) to the aasserver.json, having the entries look like this:
```
{
    "uniqueId": "myUniqueId",
    "submodelEndpoint": "http://host.docker.internal:8082/submodels/TWF0ZXJpYWxfZXhhbXBsZQ==",
    "idShortPath": "myIdShortPath",
    "api": "DotAAS-V3"
  },
```
it is working perfectly fine if the datatype coming from the source (in this case the demo opcua-server) is an integer or float, but for Strings it only works with the AAS v2 Setup for me.

As used in the DatabridgeDemo of the opc2aas package (https://github.com/eclipse-basyx/basyx-applications/tree/main/opc2aas) a jacksontransformer and a jsonatatransformer are in use, so no changes to the demo files except for the one to aasserver.json as described before and the corresponding uniqueId in routes.json were made.

One more adjustment is the use of the databridge-image: eclipsebasyx/databridge:1.0.0-milestone-07 instead of the newest one (eclipsebasyx/databridge:1.0.0-SNAPSHOT) as this is not working for me as it is permanently restarting.

Any help or idea would help a lot.

KR,
Jan

---
Further information:
The Solution mentioned in the closed issue [Error transfering string values from OPC UA server to AAS #223](https://github.com/eclipse-basyx/basyx-databridge/issues/223) to use the trigger "event" instead of "timer" makes no difference in the setup described.

**Comment by FrankSchnicke** [comment]
Hi Jan,

thank you very much for pointing out this issue. We will take a look at it and come back to you.

Best regards,
Frank

**Comment by FrankSchnicke** [comment]
This issue is fixed and included in the new 1.0.0-SNAPSHOT image on DockerHub. Thanks again for reporting this issue!
